src/styleToCssString.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * styleToCssString converts a react js style object to a css string value.
+ */
+
+import {isUnitlessNumber} from 'react/lib/CSSProperty';
+import hyphenateStyleName from 'fbjs/lib/hyphenateStyleName';

```

Comment:
Small note: If we use `fbjs` we will need to add it to package.json. Currently it can be found in `node_modules` because some our our dependencies depend on `fbjs` but we can't count on that always being true.